examples/README.md

```diff
@@ -0,0 +1,55 @@
+# Manticore Examples
+
+## Quickstart
+
+Install and try Manticore in a few shell commands:
+
+```bash
+# Install system dependencies
+sudo apt-get update && sudo apt-get install python3 python3-dev python3-pip -y
+
+# Install Manticore and its dependencies
+sudo pip3 install manticore[native]
```

Comment:
I don't think we should tell people to use `sudo`
```suggestion
pip3 install --user manticore[native]
```
Or tell them how to set up a virtual environment here.